### PR TITLE
Update fma-holocene.md

### PR DESCRIPTION
**Pull request by BlocksOnAChain**
action items for the Holocene FMA are partially closed now. Will wait for @geoknee to do one more pass and close the remaining items.

**Description**
closed some action items for the fma

**Tests**

**Additional context**

**Metadata**

### CODE REVIEW

File: security/fma-holocene.md [hunk 1]
```diff
@@ -167,19 +167,37 @@ The changes to the MIPs contract were subject to an audit, the report is availab
...
+    * TODO link to op-node/op-geth/reth activation releases and k8s PR for rollout
+
+- [x] (BLOCKING): we will re-architect the batcher to eliminate the cases above
+    * [Holocene-D: Batcher Hardening](https://github.com/ethereum-optimism/optimism/issues/12120)
+    * [batcher: keep blocks, channels and frames in strict order & simplify reorg handling #12390](https://github.com/ethereum-optimism/optimism/pull/12390)
+    * [op-batcher: add channel.straddlesActivation method (for Holocene) and wire it up #12743](https://github.com/ethereum-optimism/optimism/pull/12743)
+- [x] (BLOCKING): we will add functionality to the batcher to periodically detect when the safe chain should have progressed but didn't, and to recover from that
+    * [batcher: batchSubmitter.checkExpectedProgress #12430](https://github.com/ethereum-optimism/optimism/pull/12430)
 - [ ] (BLOCKING): the script(s) or tooling for the `SystemConfig` upgrade contract will exit with an error if the activation time has not yet passed.
+    * The scripts are here https://github.com/ethereum-optimism/optimism/pull/12878 but there is no check on the activation time
```
Comment (geoknee) [inline security/fma-holocene.md:189]: This is unresolved for now cc @clabby

File: security/fma-holocene.md [hunk 2]
```diff
@@ -167,19 +167,37 @@ The changes to the MIPs contract were subject to an audit, the report is availab
...
-- [ ] (BLOCKING): We will be testing the activation on our devnets and testnets.
+- [x] (BLOCKING): We have implemented extensive unit and end-to-end testing of the activation flow.
+    * [op-e2e/actions: Cover Holocene activation and pipeline reset #12891](https://github.com/ethereum-optimism/optimism/pull/12891)
+    * [op-node/rollup/derive: Implement pipeline stage multiplexing](https://github.com/ethereum-optimism/optimism/pull/12506/files#diff-6785b6642247a593b186581ea0e44c4ceaaac682cc45177adf059160e083efe9)
+- [x] (BLOCKING): We have implemented extensive cross-client / differential testing of the new functionality.
+    * action tests run on op-node, op-program and kona (see links above)
+- [x] (BLOCKING): We will be testing the activation on our devnets and testnets.
+    * see above
 - [ ] (BLOCKING): We have prepared datadir backups close before the upgrade, so we can use these in an emergency to rollback.
 - [ ] (BLOCKING): We have updated the runbook for recovering from a hardfork activation chain halt (including rolling back contract changes), if necessary. See https://oplabs.notion.site/RB-000-How-To-Rewind-a-Network-c21f628205354dbdbed9c691b2455a7c?pvs=74.
```
Comment (geoknee) [inline security/fma-holocene.md:202]: These items unresolved for now.

**Comment by tynes** [comment]
Wondering the status of this PR @BlocksOnAChain

File: security/fma-holocene.md (hunk 1 above)
Comment (geoknee) [reply security/fma-holocene.md:189]: In the end it was decided that it is not necessary to (needlessly cautious to) implement this check, so we can remove it from the TODOs. Instead we split the superchain ops tasks into pre and post activation, and don't execute the post activation until afterwards.

**Comment by BlocksOnAChain** [comment]
Closing this one, related to our decision that George mentioned above: "In the end it was decided that it is not necessary to (needlessly cautious to) implement this check, so we can remove it from the TODOs"
CC: @tynes for visibility.

**Comment by geoknee** [comment]
I think we still want to merge this PR, just needs a couple of edits. I can handle it @BlocksOnAChain @tynes

**Review by maurelian — APPROVED** [review]
Approving the transition to `Status: Final`, on the basis that Blocking items have been marked as complete, with links to relevant PRs and other documents.

I did not one `TODO` that should be resolved before merging this.

File: security/fma-holocene.md [hunk 3]
```diff
@@ -167,21 +167,38 @@ The changes to the MIPs contract were subject to an audit, the report is availab
...
+    * [Holocene-D: Action Tests](https://github.com/ethereum-optimism/optimism/issues/12449)
+    * [op-e2e/actions: Add Holocene FP action tests #12520](https://github.com/ethereum-optimism/optimism/pull/12520)
+    * [op-e2e/actions: Cover Holocene activation and pipeline reset #12891](https://github.com/ethereum-optimism/optimism/pull/12891)
+    * [chore(ci): Bump monorepo commit](https://github.com/anton-rs/kona/pull/805)
+
+- [x] (BLOCKING): The changes will be deployed to a devnet with both geth and reth running
+    * [Add Holocene activation timestamps for Sepoila devnets and testnets](https://github.com/ethereum-optimism/superchain-registry/pull/677)
+    * [chore: Update Holocene timestamp for Sepolia](https://github.com/paradigmxyz/reth/pull/12479)
+    * Reth does not have a chain spec built in for devnet, so a custom one will be used
+    * TODO link to op-node/op-geth/reth activation releases and k8s PR for rollout
```
[the remark this answers is not in the document]
Comment (geoknee) [reply security/fma-holocene.md]: [8012eb4](https://github.com/ethereum-optimism/design-docs/pull/160/commits/8012eb4a725a1301879a826a6c33ee80d98a28b4)

**Pull request merged**